src/main/kotlin/ar/edu/unahur/obj2/vendedores/CentroDeDistribucion.kt

```diff
@@ -0,0 +1,20 @@
+package ar.edu.unahur.obj2.vendedores
+
+class CentroDeDistribucion ( val ciudad: Ciudad ) {
+    val vendedores = mutableListOf<Vendedor>() 
+
+    fun contratarVendedor(unVendedor: Vendedor) {
+        check (vendedores.contains(unVendedor)) {
+            //El vendedor ya está contratado.
+        }
+        vendedores.add(unVendedor)
+    }
```

Comment:
Interesante... 🤔 Podrías ponerle un mensajito para que el error sea más amistoso 
```suggestion
    fun contratarVendedor(unVendedor: Vendedor) {
        check (vendedores.contains(unVendedor)) {
            "El vendedor ya está contratado"
        }
        vendedores.add(unVendedor)
    }
```